app/konnect/runtime-manager/runtime-instances/renew-certificates.md

```diff
@@ -57,6 +57,19 @@ You can generate a new data plane certificate from the {% konnect_icon runtimes
 
 1. Store the files on the local file system.
 
+Alternatively, you can generate a certificate locally and use the [pin data plane client certificate](https://developer.konghq.com/spec/3c38bff8-3b7b-4323-8e2e-690d35ef97e0/16adcd15-493a-49b2-ad53-8c73891e29bf#/DP%20Certificates/post-dp-client-certificates) endpoint to add it to Konnect
+
+```bash
+# Generate the certificate + key
+openssl req -new -x509 -nodes -newkey rsa:2048 -subj "/CN=kongdp/C=US" -keyout ./tls.key -out ./tls.crt
+
+# Get the certificate as a single line for the API call
+export CERT=$(awk 'NF {sub(/\r/, ""); printf "%s\\n",$0;}' tls.crt)
+
+# Send the certificate to Konnect
+curl https://us.api.konghq.com/v2/runtime-groups/{runtimeGroupId}/dp-client-certificates --json '{"cert":"'$CERT'"}'
+```
```

Comment:
Can you split this into two tabs, "Using Konnect" and "Generate locally"? Or h4 headings, that would also work.

Also, please split the codeblock into one for each command, using the comment as the step instruction. We get a lot of complaints about codeblocks like this, since someone can't just use the copy button for them.

(If you're fine with it, I'm happy to go ahead and make these changes)